VERSION

```diff
@@ -1,4 +1,4 @@
-VERSION="2.5.7"
+VERSION="2.5.8"
```

Comment:
Let's try to have this version bump as the last commit, the one that will be tagged, if possible :slightly_smiling_face:

Reply:
oh ok!